pkgs/tools/system/rex/default.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs, lib, fetchurl, perlPackages }:
+
+perlPackages.buildPerlPackage rec {
+  pname = "Rex";
+  version = "1.13.4";
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/F/FE/FERKI/Rex-1.13.4.tar.gz";
```

Comment:
might make sense to soft-code the version, e.g. `mirror://cpan/authors/id/F/FE/FERKI/Rex-${version}.tar.gz`